### PR TITLE
Updates for assay, platform, template

### PR DESCRIPTION
**Pull request by anngvu**
Issues somewhat related to NFOSI-186:
- [x] #176
- [x] #177
- [x] #178 -- 
- [x] #175 -- _might need to surface an extended template_

### CODE REVIEW

File: modules/Assay/Platform.csv [hunk 1]
```diff
@@ -5,44 +5,44 @@ Orbitrap Fusion Lumos Tribrid,Thermo Scientific Orbitrap Fusion Lumos Tribrid Ma
...
-HiSeq4000,Illumina HiSeq 4000,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL20301,platform,,,,HiSeq4000,Class,Assay,Platform,Platform,,,
-NextSeq500,Illumina NextSeq 500,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL18573,platform,,,,NextSeq500,Class,Assay,Platform,Platform,,,
-NextSeq 550,Illumina NextSeq 550,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL21697,platform,,,,NextSeq 550,Class,Assay,Platform,Platform,,,
-NextSeq 1000,Illumina NextSeq 1000,,,FALSE,,platform,,,,NextSeq 1000,Class,Assay,Platform,Platform,,,
-NextSeq 2000,Illumina NextSeq 2000,,,FALSE,,platform,,,,NextSeq 2000,Class,Assay,Platform,Platform,,,
-HiSeq2000,Illumina HiSeq 2000,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL11154,platform,,,,HiSeq2000,Class,Assay,Platform,Platform,,,
-MiSeq,Illumina MiSeq,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL15520,platform,,,,MiSeq,Class,Assay,Platform,Platform,,,
+Illumina HiSeq 3000,Illumina HiSeq 3000,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL21290,platform,,,,HiSeq3000,Class,Assay,Platform,Platform,,,
```
Comment (allaway) [inline modules/Assay/Platform.csv:8]: This is great! Do we need to remap values in the portal as well?

**Comment by allaway** [comment]
Re M3 MPNST: I think the name "M3 MPNST" is fine since that is the value as provided by the ctb on GEO. A description could be "A patient-derived MPNST cell line." We can also create a nf tools database listing, time permitting. Maybe once they publish their paper.

File: modules/Assay/Platform.csv (hunk 1 above)
Comment (anngvu) [reply modules/Assay/Platform.csv:8]: Yep, can track that in the internal JIRA issue.

**Review by allaway — APPROVED** [review]
LGTM, thank you!

**Comment by allaway** [comment]
## NADIA Curation Report

**Synapse Project:** [syn74318662](https://www.synapse.org/#!Synapse:syn74318662)

### Annotations Applied
| Field | Value |
|-------|-------|
| studyName | Repurposing FDA-approved drugs for the treatment of malignant peripheral nerve sheath tumors |
| diseaseFocus | Neurofibromatosis type 1 |
| manifestation | Malignant Peripheral Nerve Sheath Tumor |
| dataType | geneExpression, drugScreen |
| assay | rnaSeq |
| species | Homo sapiens |
| tumorType | Malignant Peripheral Nerve Sheath Tumor |
| platform | Illumina |
| studyLeads | Yuka Imamura Kawasawa |
| institutions | Penn State College of Medicine |
| alternateDataRepository | geo:GSE159896, insdc.sra:PRJNA670744 |

### Curation Notes
- RNA-seq of MPNST cells after FDA-approved drug treatment
- 43 FASTQ files in Raw Data folder
- No PMID found at time of curation

**Status:** pendingReview ✓

**Pull request merged**